### PR TITLE
add note that ceph.conf.template should no longer be modified

### PR DESCRIPTION
**Pull request by batrick**
Signed-off-by: Patrick Donnelly <pdonnell@redhat.com>

### CODE REVIEW

File: teuthology/ceph.conf.template [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# XXX
+#
+#   DO NOT MODIFY THIS FILE
+#
+# This file is a legacy ceph.conf template used only when testing older
+# releases of Ceph (pre-Nautilus). The new template exists in ceph.git at
+# qa/tasks/ceph.conf.template
+#
+# XXX
```
Comment (kshtsk) [inline teuthology/ceph.conf.template]: what is that XXX?

Comment (batrick) [reply teuthology/ceph.conf.template]: Many text editors show "XXX" in red highlighting to draw attention.

Comment (kshtsk) [reply teuthology/ceph.conf.template]: oh... good to know thanks

**Comment by kshtsk** [comment]
wonder, does this mean that #1308 should be closed?

**Comment by batrick** [comment]
> wonder, does this mean that #1308 should be closed?

It should be redone against the ceph.git template.

**Comment by kshtsk** [comment]
@smithfarm I guess for older releases we need to back port:
https://github.com/ceph/ceph/commit/de9b77cd38970106f41376460e0842305594f5ec
and https://github.com/ceph/ceph/commit/1ab352d

Otherwise it will still need to read this file.

**Comment by smithfarm** [comment]
@kshtsk Good idea. See https://tracker.ceph.com/issues/42261

**Pull request merged**